### PR TITLE
Annotations in preview

### PR DESCRIPTION
**Pull request by superojla**
Documentation updates for V4 annotations in content preview.

Fixes #[DDOC-647](https://jira.inside-box.net/browse/DDOC-647)

### CODE REVIEW

File: content/guides/embed/ui-elements/preview.md [hunk 1]
```diff
@@ -317,6 +317,89 @@ preview.addListener("rotate", data => {
...
+You can enable V4 [annotations][annotationsguide] in content preview. New
+annotations will sync in real time.
+
+To add V4 annotations to preview:
+
+1. Run `npm i box-annotations@latest`to install [box annotations][annotations].
+<Message warning>
+Box annotations version should be at least major version 4 and up.
+</Message>
+2. Run `npm i box-ui-elements@15.0.0` to install [BUIE][buie] version with
```
Comment (greg-in-a-box) [inline content/guides/embed/ui-elements/preview.md:331]:
```suggestion
2. Run `npm i box-ui-elements@16.0.0` to install [BUIE][buie] version with
```

We are going to release BUIE 16 to include these changes, 15 is the current major release

File: content/guides/embed/ui-elements/preview.md [hunk 2]
```diff
@@ -317,6 +317,89 @@ preview.addListener("rotate", data => {
...
+fully working V4 annotations.
+</Message>
+3. Import content preview and box annotations into your application:
+
+```js
+import ContentPreview from 'box-ui-elements/es/elements/content-preview';
+// In this example we dynamically import box-annotations which will provide a
+BoxAnnotations object in global 
+// You may see one already from legacy annotations if you do not dynamically
+the latest version of box-annotations
```
Comment (greg-in-a-box) [inline content/guides/embed/ui-elements/preview.md:344]:
```suggestion
// You may see one already from legacy annotations if you do not dynamically
load the latest version of box-annotations
```

File: content/guides/embed/ui-elements/preview.md [hunk 3]
```diff
@@ -317,6 +317,89 @@ preview.addListener("rotate", data => {
...
+                    hasActivityFeed: true, // Enabled Activity Feed which will
+                    show you the comments in the sidebar
+                    features: {
+                        activityFeed: {
+                            annotations: {
+                                enabled: true // Enables the ability to see
+                                your annotation comment in the Activity Feed 
+                            }
+                        }
+                    }
```
Comment (greg-in-a-box) [inline content/guides/embed/ui-elements/preview.md:380]: This is undocumented but we use it internally, this may or may not change in future releases

**Review by greg-in-a-box — CHANGES_REQUESTED** [review]
Left some comments

File: content/guides/embed/ui-elements/preview.md (hunk 1 above)
Comment (greg-in-a-box) [reply content/guides/embed/ui-elements/preview.md:331]: https://jira.inside-box.net/browse/DDOC-688

**Review by greg-in-a-box — CHANGES_REQUESTED** [review]
Left a couple comments, everything else looks fine

**Review by greg-in-a-box — APPROVED** [review]

**Review by bszwarc — APPROVED** [review]

**Pull request merged**